### ISSUE TITLE
 Dalen-Fuchs nodules

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#9657]
**Preferred term label:**

**Synonyms**

**Definition (free text, please give PubMed ID)**

 The epithelioid cells and giant cells often contain melanin pigment. The inflammatory process does not usually involve the choriocapillaris (in contrast to Voyt-Koyanagi-Harada syndrome) or the retina. Dalen-Fuchs nodules, which are clusters of epithelioid cells containing pigment lying between the RPE and Bruchs membrane, are also seen. Anteriorly, the iris can appear thickened with nodular infiltrations (PMID: 2228475) https://eyewiki.aao.org/Sympathetic_Ophthalmia#cite_note-16

PMID: 33256432 Dalen-Fuch nodules and depigmentation  fundus can be found in Vogt-Koyanagi-Harada syndrome (VKH). SO and VKH have similar ocular and systemic manifestations. However, patients with VKH have no history of surgery or trauma.

PMID: 35647958 Sympathetic ophthalmia is a rare, bilateral, granulomatous, panuveitis following penetrating trauma or surgery to one eye. Clinical presentation commonly occurs within the first year of trauma occurrence but can be delayed by several years. It manifests as acute/chronic granulomatous uveitis with yellowish‑white choroidal lesions or Dalen–Fuchs nodules.

(.....) Dalen (1904) described nodules on the inner surface of the choroid, whereas Ernst Fuch (1905) provided an accurate histopathological description of the disease, including non‑necrotic granulomas, lymphocytic infiltration, giant cells formation, and pigmented phagocytes.

PMID: 34283983 The classic signs of SO include conjunctival injection and granulomatous uveitis with mutton-fat keratic precipitates and choroidal Dalen-Fuchs nodules. There also may be vitritis, papillitis, retinal vasculitis, and exudative retinal detachment

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Vogt-Koyanagi-Harada syndrome ORPHA 3437
 sympathetic ophthalmia (SO) ORPHA 79098

**Your nano-attribution (ORCID)**

**Comment by carolina-f** [comment]
![1_541_2](https://user-images.githubusercontent.com/112624418/225920195-e8e2cd06-22f7-4039-afc3-f2528826231f.jpg)
![Buchs_membrane](https://user-images.githubusercontent.com/112624418/225920210-782e69bd-2e81-4de8-8d04-09fbb1dec2e0.svg)
![SO](https://user-images.githubusercontent.com/112624418/225920241-f53e8126-34bc-444b-95c1-1f17719b97de.jpg)

I am not an ophthalmology specialist, but in my opinion it's the correct parent
